### PR TITLE
Updated the documentation for tilkynna

### PR DESCRIPTION
**Pull request by melissapalmer**
Updated the documentation for tilkynna

### CODE REVIEW

File: helm-chart/tilkynna/static-files/tilkynna-realm.json [hunk 1]
```diff
@@ -0,0 +1,1597 @@
+{
```
Comment (VictorSzewczenko) [inline helm-chart/tilkynna/static-files/tilkynna-realm.json:1]: Just wondering if we still need the realm.json file?

File: quickstart/docker-compose/scripts/security-contract-template.yaml.new [hunk 2]
```diff
@@ -0,0 +1,188 @@
+applications:
```
Comment (VictorSzewczenko) [inline quickstart/docker-compose/scripts/security-contract-template.yaml.new:1]: I just noticed that this security contract has a .new extension and was wondering how it differs from the security-contract-template.yaml?

File: helm-chart/tilkynna/static-files/tilkynna-realm.json (hunk 1 above)
Comment (melissapalmer) [reply helm-chart/tilkynna/static-files/tilkynna-realm.json]: Ah helm chart should not even be on this branch :-( 
I have removed the helm-chart folder (committed in error to this branch)
https://github.com/GrindrodBank/tilkynna/pull/51/commits/0ac2f8a789d1a57ef5421625accf67aec48d93f4

File: quickstart/docker-compose/scripts/security-contract-template.yaml.new (hunk 2 above)
Comment (melissapalmer) [reply quickstart/docker-compose/scripts/security-contract-template.yaml.new]: Also committed in error :-( Removed with https://github.com/GrindrodBank/tilkynna/pull/51/commits/0a58f256d6264dcce9ef2a12d6b631e6db20a52f

**Pull request merged**